src/test/java/task40/TableSortSearchTest.java

```diff
@@ -1,47 +1,53 @@
 package task40;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.support.ui.Select;
+import task40.helpes.BaseTest;
 import task40.helpes.Employee;
 import task40.helpes.TableSortSearchPage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class TableSortSearchTest {
-    private WebDriver driver;
+import static task40.helpes.TestConstants.*;
+
+public class TableSortSearchTest extends BaseTest {
     private final int ageOption = 25;
     private final int salaryOption = 150000;
+    private final String tableCssSelector = "table#example tbody";
+    private List<Employee> customEmployeeList, expectedEmployeeList = new ArrayList<Employee>();
 
-    @BeforeEach
-    void setUp()
-    {
-        driver = new ChromeDriver();
+    public TableSortSearchTest() {
+        super(TABLE_SORT_URL);
     }
+
     @Test
     void tableSortSearchTest() throws InterruptedException {
-        List<Employee> employeeList = new ArrayList<Employee>();
-        TableSortSearchPage page = new TableSortSearchPage(driver);
+        List<Employee> actualEmployeeList = new ArrayList<Employee>();
+        TableSortSearchPage page = new TableSortSearchPage(super.driver);
         Select select = page.getWebElementByName("example_length");
         select.selectByValue("10");
 
-        page.collectTableDataList("table#example", employeeList);
+        page.collectTableDataList(tableCssSelector, actualEmployeeList);
```

Comment:
tableCssSelector - test class know anything about it

Reply:
Done